drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -172,6 +172,12 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
     block_settings_key = 'drag-and-drop-v2'
     has_score = True
 
+    def max_score(self):
+        """
+        Return the problem's max score. Required by the grading system in the LMS.
+        """
+        return self.weight
```

Comment:
Please note that `max_score` is different from `weight`.  The weight is already applied when calculating the problem's score.  So there's no need to specify it in the max_score value as well.  In fact, this may result in unexpected value.  See the formula at https://openedx.atlassian.net/wiki/display/TNL/Robust+Grades+Design#RobustGradesDesign-ProblemScores.